apps/web/src/contexts/payment-context.tsx

```diff
@@ -271,7 +271,6 @@ export function usePaymentProvider({
           .catch(async (error) => {
             const response = await error.response.json()
             mapCircleErrors(response.code)
-            setFormErrors({})
```

Comment:
This line and the one on :378 was overriding the errors that are set in `mapCircleErrors` (called on previous line). As a result, if you entered a bad CC number or full name, the form would be in an error state, but would not display the validation errors.